runtime/src/main/java/io/quarkus/reactivemessaging/http/runtime/HttpCloudEventHelper.java

```diff
@@ -1,8 +1,12 @@
 package io.quarkus.reactivemessaging.http.runtime;
 
+import static java.time.temporal.ChronoField.*;
```

Comment:
You can use our formatter as described here: https://github.com/quarkusio/quarkus/blob/main/CONTRIBUTING.md#ide-config-and-code-style

Usually we don't do star importing.

Reply:
Adjusted.